Detectors/ITSMFT/common/simulation/src/SimulationAlpide.cxx

```diff
@@ -129,21 +143,21 @@ SimulationAlpide::generateClusters(const SegmentationPixel *seg, DigitContainer
     Double_t beta = std::min(0.99999, trackP4.Beta());
     Double_t bgamma = beta / sqrt(1 - pow(beta, 2));
```

Comment:
@dr4kan  Hi Davide.  Could you please quickly commit the changes requested by Sandro ?
Essentially:  replace "pow(x,2)" with "x*x",  and "sqrt" with "TMath::Sqrt"
Then, we would quickly merge this PR.

Thanks a lot,
Iouri.